src/test/java/core/basesyntax/dao/StotageDaoTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StotageDaoTest {
+    private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
```suggestion
    private final StorageDao storageDao;
```
move initiailization to a method that is being executed before class